src/common/framework/test_config.ts

```diff
@@ -36,6 +36,11 @@ export type TestConfig = {
    * Whether to enable the `logToWebSocket` function used for out-of-band test logging.
    */
   logToWebSocket: boolean;
+
+  /**
+   * Whether to run slow failure diagnostics. In particular, computing texture sample locations.
+   */
+  verboseFailures: boolean;
```

Comment:
I think it would make sense for this to be part of the same flag as enableDebugLogs. The usage could be something like:

```ts
if (failure) {
  if (t.rec.debugging) {
    // ...
  }
}
```

or

```ts
if (failure) {
  t.debug(() => {
    // all this code runs only if debug logs are enabled
  });
}
```

I'm OK with adding a new flag if you prefer it though.

Reply:
No, it's find to use debug. Uploaded a new PR.